core/client/src/main/java/alluxio/client/AbstractOutStream.java

```diff
@@ -24,17 +24,18 @@
  */
 @NotThreadSafe
 public abstract class AbstractOutStream extends OutputStream implements Cancelable {
-  // TODO(binfan): make mBytesWritten long so we could make offset > 2GB. This
-  // requires checking the codebase for this assumption (using int to denote an offset).
-  // See ALLUXIO-1536.
   /** The number of bytes written. */
-  protected int mBytesWritten = 0;
+  protected long mBytesWritten = 0;
 
   /**
    * @return the number of bytes written to this stream
+   * @deprecated use {@link #getTotalBytesWritten()} to avoid integer overflow
```

Comment:
Oh I meant the `@deprecated` hint text, since we no longer have `getTotalBytesWritten`